### ISSUE TITLE
Docker local Dev environment

### ISSUE THREAD

**Issue by alexander-sibert** [mirumee/saleor#3019]
Maybe i do it on wrong way.

I cloned by git clone https://github.com/mirumee/saleor.git into my local folder on macOS.
I use Docker Version 18.06.1-ce-mac73 (26764)

Then i tried 
docker-compose build web
docker-compose up

But i get following error message at end:
 ---> Running in f1c03944ca9a
Warning: --system is intended to be used for pre-existing Pipfile installation, not installation of specific packages. Aborting.
ERROR: Service 'celery' failed to build: The command '/bin/sh -c pipenv install --system --deploy' returned a non-zero code: 1

**System information**
Operating system: Apple macOS 10.14 (18A391)
Browser: Mozilla Firefox 62

**Comment by maarcingebala** [comment]
Hey, it's a known bug, it was introduced with recent PRs #3018 and #3015. Until we fix it on `master` you can fix it in your local copy by changing [these lines](https://github.com/mirumee/saleor/blob/master/Dockerfile#L11-L15) to:

```
ADD requirements.txt /app/
RUN pip install --upgrade pip
RUN pip install -r /app/requirements.txt
RUN pip install -r /app/requirements_dev.txt
```

**Comment by alexander-sibert** [comment]
Thanks now i get following message: Could not open requirements file: [Errno 2] No such file or directory: '/app/requirements_dev.txt'

**Comment by maarcingebala** [comment]
My mistake - you should also add `ADD requirements_dev.txt /app/` before those `RUN` commands.

**Comment by alexander-sibert** [comment]
Thanks and then i run Docker env with: docker-compose up?

**Comment by maarcingebala** [comment]
Yep, that should work 🤞

**Comment by alexander-sibert** [comment]
It seems run with some errors:

npm WARN saleor@0.0.0 No description
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@1.2.4 (node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.2.4: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})

added 2930 packages from 1343 contributors and audited 73236 packages

**Comment by maarcingebala** [comment]
Aren't those just NPM warnings? I think they shouldn't affect the build.

**Comment by alexander-sibert** [comment]
Yep, that is what i mean, it makes no sense ;)

**Comment by joshuacox** [comment]
Does someone have a working branch?  I attempted a hotfix as described here in this thread, but I still get:

```
Step 11/39 : RUN pipenv install --system --deploy
 ---> Running in 92a9c3261287
Warning: --system is intended to be used for pre-existing Pipfile installation, not installation of specific packages. Aborting.
ERROR: Service 'celery' failed to build: The command '/bin/sh -c pipenv install --system --deploy' returned a non-zero code: 1
```

**Comment by Pacu2** [comment]
@joshuacox We are actively working on the solution, hope it will be live soon.

**Comment by joshuacox** [comment]
@Pacu2 I've got a fix in as well, I'll make a PR if it helps.

**Comment by maarcingebala** [comment]
Should be fixed by #3035
